### ISSUE TITLE
0setup and 1download download files insecurely, with --no-check-certificate

### ISSUE THREAD

**Issue by dimkr** [puppylinux-woof-CE/woof-CE#2330]

**Comment by sc0ttj** [comment]
Is this because Puppys didn't used to include the ca-cert package that puts cert stuff in /etc/ ?

..cos I think all Puppys have them now, as [something I can't remember related to SSL] needs them.. 

Maybe we can just remove that param? ..I'd be happy to follow suit in Pkg, whatever the fix/change is here..

**Comment by dimkr** [comment]
Nope, it looks like the problem is Debian. APT is configured to use HTTP, and the HTTPS certificate is bad, at least in some mirrors. I closed the PR because it breaks dpup.
